drivers/cache/cache_xtheadcmo.c

```diff
@@ -0,0 +1,243 @@
+/*
+ * Copyright (c) 2025 MASSDRIVER EI (massdriver.space)
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/init.h>
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/cache.h>
+#include <zephyr/logging/log.h>
+
+#define DT_DRV_COMPAT xuantie_xtheadcmo
+
+LOG_MODULE_REGISTER(cache_xtheadcmo, CONFIG_CACHE_LOG_LEVEL);
+
+struct cache_config {
+	uint32_t icache_line_size;
+	uint32_t dcache_line_size;
+};
+
+static struct cache_config cache_cfg = {
+	.icache_line_size = DT_INST_PROP(0, icache_line_size),
+	.dcache_line_size = DT_INST_PROP(0, dcache_line_size),
+};
+
+void __weak cache_data_enable(void)
+{
+	/* Platform specific */
+}
```

Comment:
why week, why empty ?

Reply:
Do these infos need to be in the commit or in the code?